src/components/IRPeaksPanel.tsx

```diff
@@ -1,31 +1,94 @@
-import { ValueRenderers } from '..';
+import {
+  ColumnDef,
+  useReactTable,
+  getCoreRowModel,
+  SortingState,
+  getSortedRowModel,
+} from '@tanstack/react-table';
+import { useState } from 'react';
+
+import { Table, ValueRenderers } from '..';
 
-import { Table } from './Table';
 import { IRPeak } from './context/data/DataState';
 
+interface ColumnPreferences {
+  visible?: boolean;
+  format?: (val: number | string) => string | number;
+  jpath: keyof IRPeak;
```

Comment:
I think you can keep the name `accessorKey` since `jpath` to me implies you can access deep properties, which react-table does not allow (you would have to define an accessor function instead).